src/cpu/core_dynrec/risc_ppc64le.h

```diff
@@ -542,11 +542,11 @@ static void inline gen_call_function_raw(void *func, bool fastcall = true)
 // generate a call to a function with paramcount parameters
 // note: the parameters are loaded in the architecture specific way
 // using the gen_load_param_ functions below
-static uint64_t inline gen_call_function_setup(void *func,
+static inline const uint8_t* gen_call_function_setup(void *func,
```

Comment:
Please excuse my lacking of understanding, why do we change `uint64_t` to `uint8_t`?

Reply:
The `*` after the `uint8_t` indicates the returned value is a pointer (or the memory address of) a unsigned 8-bit value.  The cache is a byte stream (unsigned 8-bits).

This is the new API return type (that other dynrec cores), and was the source of why the 64-bit PPC dynrec code build broke in the first place.